src/openms/include/OpenMS/CONCEPT/QCMS2IdentificationRate.h

```diff
@@ -0,0 +1,19 @@
+#ifndef OPENMS_CONCEPT_QCMS2IdentificationRate_H
+#define OPENMS_CONCEPT_QCMS2IdentificationRate_H
+#include <OpenMS/FORMAT/IdXMLFile.h>
+#include <OpenMS/FORMAT/MzTabFile.h>
+#include <OpenMS/FORMAT/MzTab.h>
+#include <vector>
+
+class OPENMS_DLLAPI QCMS2IdentificationRate
+{
+std::vector<std::pair<OpenMS::String,OpenMS::String>> ivec_;
+  public:
+    QCMS2IdentificationRate(std::vector<std::pair<OpenMS::String,OpenMS::String>> files):
```

Comment:
think of a cleaner interface.
As input data you need:
 - vector of idXML files/objects
 - vector of mzML files/objects
Both are expected to have the same size. If not, throw an exception.

Now, you could pass the filenames and load the files internally.
This is fine, however, imagine 20 metrics all getting similar input files and each of them loading the input and discarding it afterwards... not very effective.
However, if on the other hand the user provides 100 mzML files, then having these in memory concurrently will use more RAM than any reasonable machine has to offer (in addition, the metric knows best what data it needs. Here we only the number of scans, not the whole data).
Since this metric only needs a single mzML +idXML file at a time, the strategy can be either to pass all filenames and load only one mzML/idXML pair at a given time, or pass a single mzML/peptideIdentifiation object to the constructor and call the metric as often as needed. You can also go hybrid, and pass filenames for mzML and PeptideID objects for the ids....